response_parser.cpp

```diff
@@ -0,0 +1,64 @@
+#include "response_parser.h"
+#include <iostream>
+
+std::unique_ptr<Response> ResponseParser::Parse(std::string response) {
+	auto output_response = std::unique_ptr<Response>(new Response());
```

Comment:
Might be nice to break this function up, but there is a time constraint so don't worry too much about it